src/redux/reducers/main.ts

```diff
@@ -1,22 +1,23 @@
-import { SET_LOADING } from '../actionTypes';
+import { SET_APP_LOADING } from '../actionTypes';
 import { SetLoadingAction } from '../types';
 
 type Loading = {
```

Comment:
Maybe this should be `AppState`. You've even already defined an interface `AppLoading` that you didn't import from `../types`.
So, change the interface name to `AppState` (or make it an alias, I don't mind) or something more generic than `...Loading`.